src/ragapp/entrypoint.sh

```diff
@@ -23,5 +23,15 @@ else
     echo "Updated static files successfully!"
 fi
 
+# Copy default config to mounted volume if it is empty
+if [[ -n "$USE_VOLUME" ]]; then
```

Comment:
can't be always copy the default config if the folder is empty?

Reply:
No, it'll delete the data if the app already has some saved

Reply:
why does it delete the data? which data? if we only copy the file if the config file doesn't exist, it shouldn't do harm?

Reply:
Ah yeah, always copy the file if it's not exist would be fine.